public/marked_entities/delete.php

```diff
@@ -0,0 +1,34 @@
+<?php require_once(dirname(__FILE__)."/../../modules/ensure_logged_in.php"); ?>
+
+<?php
+
+require_once "../../modules/models/marked_entity.php";
+require_once "../../common.php";
+
+$UPDATABLE_FIELDS = ["title", "description", "due_at"];
```

Comment:
should an instructor also be able to attach a file to the marked entity? like a pdf or code file?